tests/foreman/cli/test_import.py

```diff
@@ -1338,3 +1411,114 @@ def test_reimport_enable_rh_repos_negative(self, test_data):
             )
         )
+
+    @data(*gen_import_chost_data())
+    def test_import_content_hosts_default(self, test_data):
+        """@test: Import all content hosts from
+        the predefined dataset
+
+        @feature: Import Content-host
+
+        @assert: Profiles for all Content Hosts created
+
+        """
+        tmp_dir = self.default_dataset[0]
+        files = dict(self.default_dataset[1])
+        files['content-views'] = os.path.join(
+            tmp_dir,
+            'exports/CHANNELS/export.csv',
+        )
+        files = update_csv_values(files, test_data, tmp_dir)
+        # import the prerequisities and content hosts
+        imports = import_content_hosts(files, tmp_dir)
+        for entity in imports:
+            self.assertEqual(imports[entity][0].return_code, 0)
+        # get the sat6 mapping of the imported organizations
+        imp_orgs = get_sat6_id(
+            Import.csv_to_dataset([files['users']]),
+            imports['organizations'][1]
+        )
+        # now to check whether all content hosts from csv appeared in satellite
+        for imp_org in imp_orgs:
+            self.assertNotEqual(
+                ContentHost.list({'organization-id': imp_org['sat6']}).stdout,
+                []
+            )
+
+    @data(*gen_import_chost_data())
+    def test_reimport_content_hosts_negative(self, test_data):
+        """@test: Repetitive Import of all content hosts from
+        the predefined dataset
+
+        @feature: Repetitive Import Content-host
+
+        @assert: Profiles for all Content Hosts created only once
+
+        """
+        tmp_dir = self.default_dataset[0]
+        files = dict(self.default_dataset[1])
+        files['content-views'] = os.path.join(
+            tmp_dir,
+            'exports/CHANNELS/export.csv',
+        )
+        files = update_csv_values(files, test_data, tmp_dir)
+        # import the prerequisities and content hosts
+        imports = import_content_hosts(files, tmp_dir)
+        for entity in imports:
+            self.assertEqual(imports[entity][0].return_code, 0)
+        # get the sat6 mapping of the imported organizations
+        imp_orgs = get_sat6_id(
+            Import.csv_to_dataset([files['users']]),
+            imports['organizations'][1]
+        )
+        chosts_before = [
+            ContentHost.list({'organization-id': imp_org['sat6']}).stdout
+            for imp_org in imp_orgs
+        ]
+        reimport_chosts = Import.content_host_with_tr_data({
+            u'csv-file': files['system-profiles'],
+            u'export-directory': tmp_dir,
+            u'verbose': True
+        })
+        self.assertEqual(reimport_chosts[0].return_code, 0)
+        self.assertEqual(
+            [
+                ContentHost.list({'organization-id': imp_org['sat6']}).stdout
+                for imp_org in imp_orgs
+            ],
+            chosts_before
+        )
+
+    @skip_if_bug_open('bugzilla', 1267224)
+    @data(*gen_import_chost_data())
```

Comment:
Replace this with the use of subTest(), we are going away from ddt.  Look at any of the recent ddt removal process for example.